packages/core/src/Utils/Analytics/index.ts

```diff
@@ -6,8 +6,6 @@ import { getLanguage } from '@deriv/translations';
 import { Analytics } from '@deriv-com/analytics';
 import { CountryUtils } from '@deriv-com/utils';
 
-import { MAX_MOBILE_WIDTH } from '../../Constants';
```

Comment:
[Q]: any reasons for removing this?

Reply:
No I have put it back.Not sure how it got removed